keyboards/bemeier/bmek/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE      = lite
+MOUSEKEY_ENABLE       = no
+EXTRAKEY_ENABLE       = yes
+CONSOLE_ENABLE        = no
+COMMAND_ENABLE        = no
+SLEEP_LED_ENABLE      = yes
+SPLIT_KEYBOARD        = no
+BACKLIGHT_ENABLE      = no
+NKRO_ENABLE           = yes
+RGBLIGHT_ENABLE       = yes
+AUDIO_ENABLE          = no
+LTO_ENABLE            = yes
+
+DEFAULT_FOLDER = bemeier/bmek/rev3
```

Comment:
I have a slight preference for requiring the revision to be specified when compiling, especially as you've already given that instruction in the readme.

```suggestion
```